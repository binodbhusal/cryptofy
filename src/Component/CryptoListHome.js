import React from 'react';
import {
  BrowserRouter as Router, Routes, Route, Link,
} from 'react-router-dom';
import Cryptolist from './CryptoList';
import CryptoDetailList from './CryptoDetailsList';
import styles from './CryptoListHome.module.css';
import About from './About';
import Footer from './Footer';
import ContactForm from './ContactForm';

const CryptoListHome = () => (
  <Router>

    <header className={styles.header}>
      {' '}
      <div>

        <a href="/" style={{ textDecoration: 'none' }}>
          {' '}
          <img width="40" height="40" src="https://img.icons8.com/color/48/000000/blockchain-new-logo.png" alt="blockchain-new-logo" />
          {' '}
          {' '}
          <span className={styles.cryptofy}>CRYPTOFY</span>

        </a>
        <div className={styles.aboutlink} style={{ textDecoration: 'none' }}>
          <Link to="/about" style={{ textDecoration: 'none' }}>About </Link>
          <Link to="/contact" style={{ textDecoration: 'none' }}>Contact</Link>
        </div>

        <div className={styles.incocontainer}>

          <img width="20" height="20" src="https://img.icons8.com/fluency-systems-filled/20/ffffff/microphone.png" alt="microphone" />
          <img width="20" height="20" src="https://img.icons8.com/fluency-systems-regular/20/ffffff/gear.png" alt="gear" />
        </div>
      </div>
    </header>
    <Routes>
      <Route path="/" element={<Cryptolist />} />
      <Route path="/crypto-details/:exchangeId" element={<CryptoDetailList />} />
      <Route path="/about/" element={<About />} />
      <Route path="/contact" element={<ContactForm />} />
    </Routes>
    <Footer />
  </Router>
);
export default CryptoListHome;
