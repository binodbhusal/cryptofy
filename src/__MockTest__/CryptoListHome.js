import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import Cryptolist from './CryptoList';
import CryptoDetailList from './CryptoDetailsList';

const CryptoListHome = () => (
  <Router>
    <Routes>
      <Route path="/" element={<Cryptolist />} />
      <Route path="/crypto-details/:exchangeId" element={<CryptoDetailList />} />
    </Routes>
  </Router>
);
export default CryptoListHome;
