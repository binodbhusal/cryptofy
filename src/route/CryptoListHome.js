import Cryptolist from '../Component/CryptoList';

const CryptoListHome = () => (

  <Cryptolist />
);
export default CryptoListHome;
