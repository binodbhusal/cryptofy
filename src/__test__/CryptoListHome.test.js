import React from 'react';
import { render } from '@testing-library/react';

import CryptoListHome from '../__MockTest__/CryptoListHome';

describe('Component', () => {
  test('Renders CryptoDetailList component', () => {
    const { container } = render(
      <CryptoListHome />,
    );
    expect(container).toMatchSnapshot();
  });
});
