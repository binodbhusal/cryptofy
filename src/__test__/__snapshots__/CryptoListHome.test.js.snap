// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Renders CryptoDetailList component 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
